python/mxnet/model.py

```diff
@@ -86,6 +88,8 @@ def _create_kvstore(kvstore, num_device, arg_params):
                                arg_params.values())
                 if max_size > 1024 * 1024 * 16:
                     update_on_kvstore = False
+            if kvstore == 'dist_sync_allreduce':
```

Comment:
It's not easy to merge these two update_on_kvstore into one place because these two kvstore are different type of objects.  But I will use allreduce instead of kv.type.